File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithPages/InsertEmptyPage.py
from asposepdf import Settings
from com.aspose.pdf import Document
from java.util import Date


class InsertEmptyPage:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWithPages/InsertEmptyPage/'
        
        # Open the target document
        pdf = Document(dataDir + 'input1.pdf')

        # insert a empty page in a PDF
        pdf.getPages().insert(1)

        # Save the concatenated output file (the target document)
        pdf.save(dataDir + "output.pdf")

        print "Empty page added successfully!"
 
if __name__ == '__main__':        
    InsertEmptyPage()

File: plugins/Aspose_Pdf_Java_for_PHP/src/Aspose/Pdf/WorkingWithPages/InsertEmptyPage.php
<?php

namespace Aspose\Pdf\WorkingWithPages;

use com\aspose\pdf\Document as Document;

class InsertEmptyPage {

    public static function run($dataDir=null)
    {
        # Open the target document
        $pdf = new Document($dataDir . 'input1.pdf');

        # insert a empty page in a PDF
        $pdf->getPages()->insert(1);

        # Save the concatenated output file (the target document)
        $pdf->save($dataDir . "output.pdf");

        print "Empty page added successfully!";

    }

}